src/types/widget-data/area.ts

```diff
@@ -75,4 +75,6 @@ export type AreaSeries<T = any> = BaseSeries & {
     };
     /** Options for the point markers of line in area series */
     marker?: AreaMarkerOptions;
+    /** Y-axis index (when using two axes) */
+    yAxis?: number;
```

Comment:
Should it be more narrow type like `0 | 1` ?

Reply:
in the future, the axes will need to have positioning settings so that you can split graphs.
Therefore, I think it would be better to leave  types as it is